Styles/Classes/ViewStyle.swift

```diff
@@ -66,7 +39,7 @@ public final class ViewStyle: NSObject {
                 return false
             case .borderWidth:
                 return false
-            case .roundCorners:
+            case .cornerRadius:
```

Comment:
we don't need this property anymore

Reply:
also we can now remove `applyLayout(to: )`

Reply:
I agree with apply layout, but why don't we need it anymore ? Wouldn't it become complicated to set corners on every single place ? This makes it easy to combine with other styles.